server/feeders/sensorsgenerator.js

```diff
@@ -55,27 +55,22 @@ var feedIDASSensors = function() {
 
   // Limit the number of calls to be done in parallel
   var q = async.queue(function(task, callback) {
-    idas.registerSensor(task.name, task.type)
-      .then(
-        function(response) {
-          sensorsAdded++;
-          return idas.initializeSensor(task.name, task.type)
-            .delay(1000) // This is to avoid clogging iotagent with requests.
-            .then(
-              function(response) {
-                sensorsInitialized++;
-              },
-              function(error) {
-                console.log('initializeSensor Error:', error);
-              });
-        },
-        function(error) {
-          console.log('createSensor Error:', error);
-        })
-      .done(
-        function(response) {
-          callback();
-        });
+    idas.registerSensor(task.restaurant, task.room, task.type)
+      .then(function(response) {
+        sensorsAdded++;
+        return idas.initializeSensor(task.restaurant, task.room, task.type)
+          .delay(1000) // This is to avoid clogging iotagent with requests.
```

Comment:
Without limiting the number of concurrent requests, previous versions of orion would eat all the memory of the system.  This, along with async, was used to avoid that.  The current version of orion doesn't seem to have this problem anymore, so I've removed these limits in the updated commits.